hasktorch/hasktorch.cabal

```diff
@@ -13,7 +13,8 @@ cabal-version:       >=1.10
 
 Flag disable-doctest
  Description: Disable doctest. ToDo: This flag is to avoid relocation-error of ghci for macos.
-   Default:       os(darwin)
+ Default:     False
+ Manual:      True
```

Comment:
As far as I could tell, `cabal` just doesn't support this `Default: os(darwin)` syntax.  It appears that by indenting `Default` past where it should be, this problem was masked.

By indenting `Default` the correct amount and using `os(darwin)`, I was seeing this when running `stack build`:

```console
$ stack --nix test hasktorch:doctests
Unable to parse cabal file from package /home/illabout/git/hasktorch2/hasktorch/hasktorch.cabal

- 16:17: 
unexpected Not a boolean: os
```

Reply:
funky!